### PR TITLE
set up for autobuilding prs

### PR DESCRIPTION
**Pull request by Kennric**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+PY?=python
```
Comment (ramereth) [inline Makefile]: Is this intentional?

Comment (Kennric) [inline Makefile]: inherited from the OSL and CASS pelican makefiles, but not actually used, I'll take it out

**Comment by Kennric** [comment]
@ramereth OK I think this is good now. http://beaver-barcamp-pelican-27.staging.osuosl.org/

**Comment by Kennric** [comment]
Oops, no it's not done, missed an import....

**Comment by ramereth** [comment]
@Kennric is there a reason why the currently built site has issues?

**Comment by Kennric** [comment]
@ramereth Yes, an http/https site url mismatch, I forgot to rebuild after making a small change to the job. It is correct now.

**Pull request merged**